src/styles/card.scss

```diff
@@ -0,0 +1,148 @@
+.main {
+  display: flex;
+  justify-content: center;
+  padding-top: 50px;
+}
+
+.main__container {
+  width: 944px;
+  height: 856px;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.main__block1 {
+  margin-bottom: 50px;
+  width: 944px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.main__block2 {
```

Comment:
if there are 3 rows or cards, will you make another class .main__block3 with the same styles?)

Reply:
delete all block